app/konnect/org-management/oidc-idp.md

```diff
@@ -59,3 +59,5 @@ You can configure custom IdP-specific behaviors in the **Advanced Settings** of
     - `name`: Used as the {{site.konnect_short_name}} account's `full_name`.
     - `email`: Used as the {{site.konnect_short_name}} account's `email`.
     - `groups`: Used to map users to teams defined in the team mappings upon login.
+   {:.note}
+> **Note:** If users are assigned a very large number of groups (over 150 in most cases), the IdP may send the groups claim in a non-standard manner which will cause authentication issues. To work around this limitation at the IdP, it is recommended to use group filtering functions provided by the IdP for this purpose. Quick reference guides: [Azure group filtering](https://learn.microsoft.com/en-us/azure/active-directory/hybrid/connect/how-to-connect-fed-group-claims#group-filtering) and [Okta group filtering](https://support.okta.com/help/s/article/How-to-send-certain-groups-that-the-user-is-assigned-to-in-one-Group-attribute-statement)https://support.okta.com/help/s/article/How-to-send-certain-groups-that-the-user-is-assigned-to-in-one-Group-attribute-statement. You may need to contact the Support team of your Identity Provider in order to see how to filter groups emitted for the application.
```

Comment:
```suggestion

## Troubleshooting 

### Authentication issues with large numbers of groups

If users are assigned a very large number of groups (over 150 in most cases), the IdP may send the groups claim in a non-standard manner, causing authentication issues. 

To work around this limitation at the IdP, we recommend using group filtering functions provided by the IdP for this purpose. 
Here are some quick reference guides for common IdPs:
* [Azure group filtering](https://learn.microsoft.com/en-us/azure/active-directory/hybrid/connect/how-to-connect-fed-group-claims#group-filtering) 
* [Okta group filtering](https://support.okta.com/help/s/article/How-to-send-certain-groups-that-the-user-is-assigned-to-in-one-Group-attribute-statement)

You may need to contact the support team of your identity provider in order to learn how to filter groups emitted for the application.
```

I think it's much more readable to add a troubleshooting section here and format the info into separate paragraphs.